airflow/runtimes/container_runtime_test.go

```diff
@@ -11,6 +11,17 @@ import (
 	"github.com/stretchr/testify/suite"
 )
 
+// mockFileChecker is a mock implementation of FileChecker for tests in this file.
+type mockFileChecker struct {
+	ExistingFiles map[string]bool
+}
+
+// FileExists is just a mock for os.Stat(). In our test implementation, we just check
+// if the file exists in the list of mocked files for a given test.
+func (m mockFileChecker) FileExists(path string) bool {
+	return m.ExistingFiles[path]
+}
+
```

Comment:
Moved this special mock here to live with its specific tests. I've also configured mockery to now generate a more standard mock under `mocks` for the other tests.